packages/ember-data/tests/integration/serializers/embedded-records-mixin-test.js

```diff
@@ -609,6 +609,31 @@ test("serialize with embedded objects (hasMany relationship)", function() {
   });
 });
 
+test("serialize with embedded objects (unknown hasMany relationship)", function() {
+  var league;
+  run(function() {
+    league = env.store.push(HomePlanet, { name: "Villain League", id: "123" });
+  });
+
+  env.registry.register('serializer:home-planet', DS.ActiveModelSerializer.extend(DS.EmbeddedRecordsMixin, {
+    attrs: {
+      villains: { embedded: 'always' }
+    }
+  }));
+
+  var serializer, json;
+  run(function() {
+    serializer = env.container.lookup("serializer:home-planet");
+
+    json = serializer.serialize(league._createSnapshot());
+  });
+
+  deepEqual(json, {
+    name: "Villain League",
+    villains: []
```

Comment:
Is this really what we want? (since the relationship is "unknown")

Excluding unknown relationships from the payload might not be correct when it comes to embedded since it _should_ be present, correct?

ping @igorT @jakesjews